pkg/evaluator/builtin.go

```diff
@@ -21,30 +21,27 @@ type Builtins struct {
 	Print         func(s string)
```

Comment:
It does not look like `Evaluator.print` is used any more, and this `Builtins.Print` is only used to initialise `Evaluator.print`. I think both of these can be removed.

Reply:
ah cool, right you are, thanks, removed.